README.md

```diff
@@ -6,18 +6,17 @@
 
 ## How can I use this?
 
-Iris is alpha quality software and is highly incomplete. If you are not a developer or familiar with compiling and running Minecraft mods from source, OptiFine is likely the better choice in the immediate moment.
+**You can download Iris from [GitHub Releases](https://github.com/IrisShaders/Iris/releases), [Modrinth](https://modrinth.com/mod/iris) or [Curseforge]().**
```

Comment:
I couldn't find the URL for the Iris curseforge page, so if someone could link it below that would be nice...

Reply:
https://www.curseforge.com/minecraft/mc-mods/irisshaders
It should be this I think.

Reply:
Thanks